ingestion/src/metadata/ingestion/source/dashboard/looker.py

```diff
@@ -235,15 +297,53 @@ def yield_dashboard_lineage_details(
                     f"Error building lineage for database service [{db_service_name}]: {err}"
                 )
 
+    def build_lineage_request(
+        self, source: str, db_service_name: str, to_entity: MetadataDashboard
+    ) -> Optional[AddLineageRequest]:
+        """
+        Once we have a list of origin data sources, check their components
+        and build the lineage request.
+
+        We will try searching in ES with and without the `database`
+
+        Args:
+            source: table name from the source list
+            db_service_name: name of the service from the config
+            to_entity: Dashboard Entity being used
+        """
+
+        source_elements = fqn.split_table_name(table_name=source)
+
+        for database_name in [source_elements["database"], None]:
```

Comment:
build the FQN using the given database name or without it. We return the first match